app/src/components/dialog/InfoDialog.tsx

```diff
@@ -55,7 +55,7 @@ const InfoDialog: React.FC<IInfoDialogProps> = (props) => {
   };
 
   const handleError = (error: any) => {
-    console.error('Error playing media', error);
+    console.error('Unable to play media', error);
```

Comment:
should probably remove console log

Reply:
It is there so we know which paths in the S3 are missing, probably we will remove once we figure out our interactions with the bcGov docs team as they will be providing content